edx_rest_framework_extensions/auth/jwt/cookies.py

```diff
@@ -19,3 +20,17 @@ def jwt_cookie_signature_name():
 
 def jwt_refresh_cookie_name():
     return settings.JWT_AUTH.get('JWT_AUTH_REFRESH_COOKIE') or 'edx-jwt-refresh-cookie'
+
+
+def get_decoded_jwt_from_cookie(request):
+    """
+    Grab jwt from request cookie if possible.
```

Comment:
Nit: update the docstring to indicate which cookie is being parsed.

Reply:
done.